plugins/services/src/js/service-configuration/NetworkingServiceConfigSection.js

```diff
@@ -100,10 +101,19 @@ module.exports = {
           }
         ];
 
+        if (editLink) {
+          columns.push({
+            heading() { return null; },
+            className: 'configuration-map-action',
+            prop: 'edit',
+            render() { return editLink; }
```

Comment:
The same issue with the table as I mentioned before. (Just repeating to help you track occurrences)